hotdoc/extensions/gst/gst_extension.py

```diff
@@ -643,23 +639,36 @@ def _get_smart_key(self, symbol):
 
         return res
 
+    def __create_symbol(self, gtype, symbol, pagename, other_types_pages):
+        if symbol["kind"] in ["enum", "flags"]:
+            self.__create_enum_symbol(gtype, symbol.get('values'), pagename)
+        elif symbol["kind"] == "object":
+            self.__create_object_type(pagename, symbol, other_types_pages)
+
+    @staticmethod
+    def _remember_symbol_type(gtype, pagename, other_types_pages):
```

Comment:
Can you make `other_types_pages` an instance attribute instead, and document its purpose? :)

Reply:
We should keep it as small as possible, and thus get it reset on each plugin, I find it cleaner this way.

Reply:
That's why I said instance attribute, don't we have one extension instance per plugin?